src/include/cephd/libcephd.h

```diff
@@ -0,0 +1,48 @@
+#ifndef CEPH_LIBCEPHD_H
+#define CEPH_LIBCEPHD_H
```

Comment:
librados, librbd and libcephfs all use old style header guards. I'm tempted to leave this as is for consistency. I like the pragma once as well :-) Let me know if you feel strongly about this one.